build/scripts/build.ps1

```diff
@@ -94,7 +94,7 @@ function Run-MSBuild([string]$buildArgs = "", [string]$logFile = "") {
     }
 
     $args += " $buildArgs"
-    Exec-Command $msbuild $args
+    Exec-Console $msbuild $args
```

Comment:
Believe so yes. But should remove all of the `Out-Host` pipes from calls to Run-MSBuild